<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTerminologyNavigatorWidget</class>
 <widget class="qMRMLWidget" name="qSlicerTerminologyNavigatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qSlicerTerminologyNavigatorWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="1" colspan="2">
    <widget class="ctkComboBox" name="ComboBox_Terminology">
     <property name="defaultText">
      <string>Select terminology...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QFrame" name="frame_TerminologyEntry">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="1" column="1">
       <widget class="QTableWidget" name="tableWidget_Type">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column/>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTableWidget" name="tableWidget_Category">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column/>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="ctkSearchBox" name="SearchBox_Category">
        <property name="placeholderText">
         <string>Category...</string>
        </property>
        <property name="showSearchIcon">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkSearchBox" name="SearchBox_Type">
        <property name="placeholderText">
         <string>Property type...</string>
        </property>
        <property name="showSearchIcon">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="ctkComboBox" name="ComboBox_TypeModifier">
        <property name="defaultText">
         <string>Select type modifier...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ctkColorPickerButton" name="ColorPickerButton_RecommendedRGB">
        <property name="displayColorName">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="ctkCollapsibleGroupBox" name="CollapsibleGroupBox_AnatomicRegion">
        <property name="title">
         <string>Anatomic region (optional)</string>
        </property>
        <property name="collapsed">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="margin">
          <number>4</number>
         </property>
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="3" column="0">
          <widget class="ctkComboBox" name="ComboBox_AnatomicRegionModifier">
           <property name="defaultText">
            <string>Select anatomic region modifier...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="ctkSearchBox" name="SearchBox_AnatomicRegion">
           <property name="placeholderText">
            <string>Anatomic region...</string>
           </property>
           <property name="showSearchIcon">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="ctkComboBox" name="ComboBox_AnatomicContext">
           <property name="defaultText">
            <string>Select anatomic region context...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QTableWidget" name="tableWidget_AnatomicRegion">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="columnCount">
            <number>1</number>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSearchBox</class>
   <extends>QLineEdit</extends>
   <header>ctkSearchBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
